rueidisaside/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/redis/rueidis/rueidisaside
+
+go 1.22.0
+
+toolchain go1.24.1
```

Comment:
```suggestion
```
We don’t use the toolchain directive in other mod files. Could you also help delete https://github.com/redis/rueidis/blob/696b5648b60bd6971e0f0d39f6c6ed964059bccd/rueidislimiter/go.mod#L5?

Reply:
Sure not problem